src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+
+  --header-height: 60px;
+  --pseudo-link-color: #00acdc;
```

Comment:
Consider moving the CSS variables (`--header-height` and `--pseudo-link-color`) to the `:root` selector instead of the `body` selector. This will make them available globally and not just within the `body` element, which is a more standard practice.